test/test_matrixroomhandler.ts

```diff
@@ -87,7 +87,7 @@ function createRH(opts: any = {}) {
         },
     };
     const us = {
-        EnsureJoin: async () => { },
+        JoinRoom: async () => { USERSJOINED++; },
```

Comment:
Are we testing this?

Reply:
Yes, implicitly in `joinRoom` of `matrixroomhandler`